modules/common/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/common/transformation/EventUtils.scala

```diff
@@ -93,6 +94,27 @@ object EventUtils {
     tabular.mkString("\t")
   }
 
+  def alterEnrichedEventAny[F[_]: Clock: RegistryLookup: Monad](originalLine: Event, lengths: Map[String, Int]): List[Any] = {
+    def transformDate(s: String): Timestamp =
+      Timestamp.from(Instant.parse(s))
+    def truncate(key: String, value: String): String =
```

Comment:
Do we still need this truncation here?  I thought in newer releases we already validate input events against the atomic event schema.

Reply:
Ah, I see we still use scala analytics sdk version 2.1.0, whereas the feature I'm talking about was added in 3.0.0.  Is there any reason we are not yet using scala analytics sdk 3.0.0?  I was expecting that change to happen for the snowflake-loader release.